tests/test_exposure.py

```diff
@@ -843,6 +843,7 @@ def setUp(self):
         self.photoCalib = afwImage.PhotoCalib(1.5)
         self.psf = DummyPsf(2.0)
         self.detector = DetectorWrapper().detector
+        self.summary = afwImage.ExposureSummary()
```

Comment:
I think the tests will be a bit more meaningful if this is not the default value. It looks like `ExposureSummary`'s constructor can take keyword arguments?